packages/core/src/routes/connector.ts

```diff
@@ -59,6 +67,57 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
     }
   );
 
+  router.post(
+    '/connectors',
+    koaGuard({
+      body: Connectors.createGuard.omit({
+        id: true,
+        enabled: true,
+        createdAt: true,
+      }),
+    }),
+    async (ctx, next) => {
+      const {
+        body: { connectorId, config, metadata },
+        body,
+      } = ctx.guard;
+
+      const standardConnectors = await loadConnectors();
+      const standardConnector = standardConnectors.find(
+        ({ metadata: { id } }) => id === connectorId
+      );
+
+      if (!standardConnector) {
+        throw new RequestError({
+          code: 'connector.invalid_connector_id',
+          status: 422,
+        });
+      }
+
+      const { count } = await countConnectorByConnectorId(connectorId);
+      assertThat(
+        count === 0 || standardConnector.metadata.isStandard === true,
+        new RequestError({
+          code: 'connector.multiple_instances_not_supported',
+          status: 422,
+        })
+      );
+
+      assertThat(!config || !metadata, 'connector.config_and_metadata_empty');
+
+      if (config) {
```

Comment:
I think we can remove "config" in this API endpoint for now. "metadata" is enough to create a "old version connector".